solidity/contracts/token/libs/MovableCollateralRouter.sol

```diff
@@ -67,44 +35,79 @@ abstract contract MovableCollateralRouter is Router {
         _;
     }
 
-    function addRebalancer(address rebalancer) external onlyOwner {
-        _allowedRebalancers.set(rebalancer, 1);
+    modifier onlyAllowedBridge(uint32 domain, ValueTransferBridge bridge) {
+        EnumerableSet.AddressSet storage bridges = _allowedBridges[domain];
+        require(bridges.contains(address(bridge)), "MCR: Not allowed bridge");
+        _;
     }
 
-    function removeRebalancer(address rebalancer) external onlyOwner {
-        _allowedRebalancers.remove(rebalancer);
+    function rebalancers() external view returns (address[] memory) {
+        return _allowedRebalancers.values();
     }
 
-    function addRebalancers(address[] calldata rebalancers) external onlyOwner {
-        for (uint256 i = 0; i < rebalancers.length; i++) {
-            _allowedRebalancers.set(rebalancers[i], 1);
-        }
+    function allowedBridges(
+        uint32 domain
+    ) external view returns (address[] memory) {
+        return _allowedBridges[domain].values();
+    }
+
+    function setRecipient(uint32 domain, bytes32 recipient) external onlyOwner {
+        _mustHaveRemoteRouter(domain);
```

Comment:
To clarify, the benefit here is just that there is only one set of domains to enumerate, correct? A comment would be nice.